### PR TITLE
scikit-surprise 1.1.3

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/NicolasHug/Surprise/blob/v1.1.3/CHANGELOG.md
License: https://github.com/NicolasHug/Surprise/blob/v1.1.3/LICENSE.md
Requirements: https://github.com/NicolasHug/Surprise/blob/v1.1.3/setup.py

Actions:
1. Fix numpy pinnings for host and run
2. Reset build number
3. Skip py<38
4. Update script
5. Add entry_points
6. Add the missing wheel to host
7. Add pip check
8. Add test commands
9. Fix home & doc urls
10. Add description
11. Skip lints

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#upload_channels:
+#  - sfe1ed40
+
+#channels:
+#  - sfe1ed40
+
```
Comment (M-Waszkiewicz-Anaconda) [inline abs.yaml]: this file can be removed ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 56bfec410b21278807b7538acb7405db99b44937259dd329762cee7e667d4bcc
 
 build:
-  number: 1
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:15]: if we haven't change package version why we reset build number ?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - surprise = surprise.__main__:main
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
-    - setuptools
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:23]: is in't setuptools mandatory in host section, linter would not complain ?

File: abs.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply abs.yaml]: No, it will be enabled after approvals

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:15]: It's a new package

File: recipe/meta.yaml (hunk 3 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:23]: The linter complains about the `run` section. In the host section, it's placed below https://github.com/AnacondaRecipes/scikit-surprise-feedstock/pull/2/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL27

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:23]: remove from run and will be ok for me

Comment (skupr-anaconda) [reply recipe/meta.yaml:23]: setuptools is required https://github.com/AnacondaRecipes/scikit-surprise-feedstock/pull/2/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR37

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by akabanovs — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,34 +1,41 @@
...
-  sha256: {{ sha256 }}
+  sha256: 56bfec410b21278807b7538acb7405db99b44937259dd329762cee7e667d4bcc
 
 build:
-  number: 1
-  script: python setup.py install --single-version-externally-managed --record record.txt
-
+  number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (sumit0190) [inline recipe/meta.yaml]: Can you also use `--no-build-isolation` here?

**Review by sumit0190 — APPROVED** [review]
LGTM with the `--no-build-isolation` change.

File: recipe/meta.yaml (hunk 4 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Added. Thanks!

**Pull request merged**